src/app/globals.css

```diff
@@ -1,33 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200");
```

Comment:
When requesting a google font, the user's ip address is sent within that request. Unfortunately, this is against the GDPR and we would be violating it. I know we are making this very same thing on other platforms and we should improve it on those too.

My suggestion is to download the font and reference it locally. This way we would be compliant with the GDPR.

Learn more [here](https://www.cookieyes.com/documentation/google-fonts-and-gdpr/).

Besides that, what a great work. Keep up with it 🙌

Reply:
Fixed! Thanks for the heads up! 🙏